### PR TITLE
Add proper link text

### PR DESCRIPTION
**Pull request by paususe**
Add proper link text

### CODE REVIEW

File: modules/installation/pages/proxy-registration.adoc [hunk 1]
```diff
@@ -51,14 +51,14 @@ Using Salt is the default.
 // Enable Remote Configuration
 // Enable Remote Commands
 For more information about bootstrap scripts, see
-xref:client-configuration:registration-bootstrap.adoc[].
+xref:client-configuration:registration-bootstrap.adoc[Registering Clients with a Bootstrap Script].
```
Comment (Loquacity) [inline modules/installation/pages/proxy-registration.adoc]:
```suggestion
xref:client-configuration:registration-bootstrap.adoc[].
```

File: modules/installation/pages/proxy-registration.adoc [hunk 2]
```diff
@@ -51,14 +51,14 @@ Using Salt is the default.
...
 +
 
 .Modifying Bootstrap Script
 image::proxy-bootstrap.png[]
 
 . Create the SUSE Manager Tools Repository for bootstrapping, see xref:client-configuration:creating-a-tools-repository.adoc[Create Tools Repository].
-. Bootstrap the client with the bootstrap script.  For more information, see xref:client-configuration/pages/registration-bootstrap.adoc[].
+. Bootstrap the client with the bootstrap script.  For more information, see xref:client-configuration:registration-bootstrap.adoc[Registering Clients with a Bootstrap Script].
```
Comment (Loquacity) [inline modules/installation/pages/proxy-registration.adoc]:
```suggestion
. Bootstrap the client with the bootstrap script.  For more information, see xref:client-configuration:registration-bootstrap.adoc[].
```

**Comment by paususe** [comment]
Ok

**Comment by Loquacity** [comment]
@paususe are you going to merge this?

**Pull request merged**